common/xrdp_client_info.h

```diff
@@ -148,6 +148,8 @@ struct xrdp_client_info
 
   int client_os_major;
   int client_os_minor;
+
+  int no_orders_supported;
```

Comment:
I wonder if `painter_needed` or `painter_used` would be a better name. Does it need to be in `struct xrdp_client_info`? That structure is becoming a "god object" that knows everything.